tests/lib/nested.sh

```diff
@@ -522,12 +522,13 @@ start_nested_core_vm_unit(){
             mv /etc/apt/sources.list.back /etc/apt/sources.list
             apt update
         fi
-
+        # use efi by default
+        PARAM_BIOS="-bios /usr/share/ovmf/OVMF.fd"
```

Comment:
hmm should this be 

```suggestion
        PARAM_BIOS="-bios /usr/share/OVMF/OVMF_CODE.fd"
```

Reply:
(I have the /usr/share/ovmf/OVMF.fd file, but I have never booted with it, I have always used the /usr/share/OVMF/OVMF_* files)

Reply:
hmm to make it more confusing:
```

google-nested:ubuntu-20.04-64 /root# dpkg -S /usr/share/ovmf/OVMF.fd 
ovmf: /usr/share/ovmf/OVMF.fd
google-nested:ubuntu-20.04-64 /root# dpkg -S /usr/share/OVMF/OVMF_CODE.fd 
ovmf: /usr/share/OVMF/OVMF_CODE.fd
google-nested:ubuntu-20.04-64 /root# md5sum /usr/share/OVMF/OVMF_CODE.fd 
fdaa1d4826a1a4de59a2eee921bb1894  /usr/share/OVMF/OVMF_CODE.fd
google-nested:ubuntu-20.04-64 /root# md5sum /usr/share/ovmf/OVMF.fd
0afa7cdae7023d4006f34b7ae6f22191  /usr/share/ovmf/OVMF.fd
```

I've always used /usr/share/ovmf/OVMF.fd, any clue what the difference might be?

Reply:
https://github.com/tianocore/edk2/commit/1c50db8adaf9d5ce071e27a518a46cd363ac5efe indicates that OVMF.fd is a bundle of OVMF_CORE.fd and OVMF_VARS.fd.

Reply:
The original reason for the CODE/VARS split was, IIRC, to allow the OVMF code to be updated while keeping the variable state for existing VMs.